Formula/n/node@16.rb

```diff
@@ -22,7 +23,6 @@ class NodeAT16 < Formula
   deprecate! date: "2023-11-02", because: :unsupported
 
   depends_on "pkg-config" => :build
-  depends_on "python-setuptools" => :build
```

Comment:
still need it?

Reply:
Oops, didn't work -- trying a different approach.